frontend/src/components/HeaderTopBar.js

```diff
@@ -103,10 +107,13 @@ class HeaderTopBar extends React.Component {
             id="header-contribute-dropdown"
             pullRight
           >
-            <MenuItem eventKey={0} onSelect={this.onSubmitYourOperator}>
+            <MenuItem eventKey={0} onSelect={this.onPreviewYourOperator}>
+              Preview your Operator
+            </MenuItem>
+            <MenuItem eventKey={1} onSelect={this.onSubmitYourOperator}>
               Submit your Operator
             </MenuItem>
-            <MenuItem eventKey={1} onSelect={this.onGettingStarted}>
+            <MenuItem eventKey={2} onSelect={this.onGettingStarted}>
```

Comment:
Can you order the menu like the order the steps would be done: 

Create
Preview
Submit